test/functional/p2p_eviction.py

```diff
@@ -122,6 +122,36 @@ def run_test(self):
         self.log.debug("{} protected peers: {}".format(len(protected_peers), protected_peers))
         assert evicted_peers[0] not in protected_peers
 
+        self.log.info("Test that whitebind inbounds get extra eviction power")
+        # Only allow 1 inbound connection, but set whitebind
+        self.restart_node(0, extra_args=['-maxconnections=12', '-whitebind=0.0.0.0:30201'])
```

Comment:
As you [mentioned](https://bitcoincore.reviews/27600#l-165) in the review club, whitelisting just based on ports enables an attacker that discovered which port(s) are whitelisted to take over all (non-whitelisted) inbound connections slots.

Therefore, I think we have to be careful not to accidentally lead people to this footgun. Binding to `127.0.0.1` seems much more prudent to not set a bad example.

```suggestion
        self.restart_node(0, extra_args=['-maxconnections=12', '-whitebind=127.0.0.1:30201'])
```

Reply:
updated test and also included warning in release note